README.md

```diff
@@ -853,6 +854,75 @@ var timestamp = time.ZonedDateTime.now().plusMinutes(5);
 console.log(timestamp.getMillisFromNow());
 ```
 
+### Quantity
+
+The `Quantity` class greatly simplifies Units of Measurement (UoM)/Quantity handling by providing unit conversion, comparisons and mathematical operations.
+
+Internally using the openHAB `QuantityType`, which relies on [`javax.measure`](https://unitsofmeasurement.github.io/unit-api/), it supports all units and dimensions that openHAB supports - see [Units of Measurement documentation](https://www.openhab.org/docs/concepts/units-of-measurement.html#list-of-units) for an (incomplete) overview.
+If you unit is not listed in the UoM docs, it is very likely that it is still supported, e.g. the Angstrom Å for very small lengths (1 Å = 10 nm).
+
+#### Creation
+
+`Quantity(value)` can be used as a factory (without `new` - don't use it as a constructor), pass an amount **and** a unit to it to create a new `Quantity` object:
+
```

Comment:
Please explain how the actual needs to be transformed, i.e. Nm -> N*m, m²/s² -> m^2/s^2, -> whitespace not allowed as I think it is not quite obvious at least that a * has to be used in Nm.

I would recommend to take some "special" examples from here https://www.openhab.org/docs/concepts/units-of-measurement.html#list-of-units and show how they would be transformed (eg. ɡₙ, °dH, W/m², µW/cm²)

Reply:
I'm cautious about the "(incomplete)" part. It's incomplete now but will it be forever? Once it becomes complete, will we remember to come and remove that here? 

I'd leave it out and create an issue to go make the linked to page complete.

Doesn't the link to the javax.measure list *all* the units? So while our docs only show the most commonly used ones in a home automation context, they can get to the complete list from there, right?

> `Quantity(value)` can be used as a factory (without `new` - don't use it as a constructor)

For someone who doesn't know programming, this sentence is confusing and scary. I'd switch it around and put the "without `new` outside the parenthetical and put "factory" inside.

>`Quantity(value)` is used without `new` (it's a factory, not a construcor).

I left out the "can" because as I understand it, this is a *must*, not a *can*.

Reply:
> Doesn't the link to the javax.measure list all the units? So while our docs only show the most commonly used ones in a home automation context, they can get to the complete list from there, right?

Unfortunately no -- I haven't found a list of the units supported by javax.measure. Even not in the JavaDoc.

Reply:
> show how they would be transformed (eg. ɡₙ, °dH, W/m², µW/cm²)

Actutally they work all except the standard gravity, I have no idea how to get that work.